server/spec/mutations/stacks/create_spec.rb

```diff
@@ -300,6 +300,29 @@
       expect(outcome.errors.message).to eq 'services' => { 'foo' => {'name' => "Create failed"}, 'bar' => { 'links' => "Service soome-stack/foo does not exist"}}
     end
 
+    pending 'reports service error array outcomes' do
```

Comment:
This is pending the fixes in #2058 for handling `Mutations::ErrorArray` in `handle_service_outcome_errors`:

```
  1) Stacks::Create#run reports service error array outcomes
     Failure/Error: outcome = described_class.new(
     ArgumentError:
       Invalid kind
     # ./vendor/bundle/ruby/2.3.0/gems/mutations-0.7.2/lib/mutations/command.rb:113:in `add_error'
     # ./app/mutations/stacks/common.rb:13:in `add_service_error'
     # ./app/mutations/stacks/common.rb:21:in `block in handle_service_outcome_errors'
     # ./app/mutations/stacks/common.rb:20:in `each'
     # ./app/mutations/stacks/common.rb:20:in `handle_service_outcome_errors'
     # ./app/mutations/stacks/create.rb:35:in `block in validate_services'
     # ./app/mutations/stacks/create.rb:29:in `each'
     # ./app/mutations/stacks/create.rb:29:in `validate_services'
     # ./app/mutations/stacks/create.rb:25:in `validate'
     # ./vendor/bundle/ruby/2.3.0/gems/mutations-0.7.2/lib/mutations/command.rb:67:in `initialize'
     # ./spec/mutations/stacks/create_spec.rb:312:in `new'
     # ./spec/mutations/stacks/create_spec.rb:312:in `block (4 levels) in <top (required)>'
     # ./spec/mutations/stacks/create_spec.rb:311:in `block (3 levels) in <top (required)>'
```